eng/performance/benchmark_jobs.yml

```diff
@@ -43,7 +43,7 @@ jobs:
             value: '.NET Performance - $(BenchViewRunType) - $(Build.SourceBranchName) $(Build.SourceVersion)'
           # for private runs we want to generate data (--generate-benchview-data ) and upload it (--upload-to-benchview-container specifies where)
           - name: BenchViewArguments
-            value: '--benchview-submission-name "$(BenchviewCommitName)" --benchview-machinepool perfsnake --generate-benchview-data --benchview-run-type $(BenchViewRunType) --upload-to-benchview-container $(_Category)'
+            value: '--benchview-submission-name "$(BenchviewCommitName)" --benchview-machinepool perfsnake --generate-benchview-data --benchview-run-type $(BenchViewRunType) --upload-to-benchview-container all'
```

Comment:
@jorive : here I have changed the name of the `--upload-to-benchview-container` from `coreclr` or `corefx` to `all`. Is this OK?

Reply:
I'd worry about that as we do not currently have an "all" container in the storage account

Reply:
@adamsitnik coreclr and corefx are different containers/blob storages with their own SAS tokens. You cannot add any random name.

Reply:
Maybe it makes sense having a single blob for all repos? Personally, I always felt it was more manageable having different blobs per repo/feature teams.

Reply:
What's the value prop for having the blobs be separate? 

I think from an execution and uploading standpoint, they should be the same. I think it's useful to have all the results for a single "run" be the same, and then have filters in the reporting if folks want to exclude stuff. I think it's *hugely* useful if by default results for CoreFX tests show up for a CoreCLR dev and vice versa - it's possible that those tests get affected by some change and knowing that ahead of time is great.

Reply:
Sorry, maybe I did not explain myself properly. How many blobs we use to upload data has nothing to do with grouping and displaying all benchmarks together. It is about managing how much data is generated by different repos with the different permutations on how the benchmarks run, and know how much we spend on Azure storage. It's for monitoring, but there are other ways you could do so.

Reply:
For the record, it is possible to have the same SAS token for different blobs, as that is what we are doing out of the perf repo (we push to both the coreclr blob/container and the corefx blob/container with the same SAS token).